### ISSUE TITLE
Files missing from Theme folder?

### ISSUE THREAD

**Issue by Kidkie** [olefredrik/FoundationPress#459]
Where are the Components folder? And JS-files?

**Comment by emcniece** [comment]
Did you figure it out? I just did a fresh install and ran into the same thing, wtf...

**Comment by Kidkie** [comment]
Maybe we are missing some bit of information? Are we supposed to get the files from somewhere ourself? Because they're not in the repo at all.

Mvh,

/ Martin

**Comment by AdamChlan** [comment]
Are the files you are looking for in the Assets folder?

**Comment by olefredrik** [comment]
Hi @emcniece & @Kidkie,

The files you are referring to are added to [.gitignore](https://github.com/olefredrik/FoundationPress/blob/master/.gitignore). It is simply because it provides no purpose in having these files under source control, since they are automatically generated by ```npm run build```. 

Given that you've run ´´´npm install´´´ after cloning the repository, all your components and scripts should be placed in your assets folder.

**Comment by emcniece** [comment]
Got it - bad root habits.

    $ npm run build: 

    > foundationpress@1.4.0 build /var/www/clients/client0/web42/web/wp-content/themes/FoundationPress
    > grunt build

    Running "copy:scripts" (copy) task


    Running "copy:iconfonts" (copy) task


    Running "string-replace:fontawesome" (string-replace) task

    Running "sass:dist" (sass) task
    >> file to import not found or unreadable: ../components/foundation/scss/normalize
    >> Current dir: /var/www/clients/client0/web42/web/wp-content/themes/FoundationPress/assets/scss/
    >>   Line 4  Column 9  assets/scss/foundation.scss
    Warning:  Use --force to continue.

    Aborted due to warnings.


    Execution Time (2015-09-23 19:53:37 UTC)
    loading tasks  729ms  âââââââââââââââââââââââââââââââââââââââââââââââââââââââââââââââââââââââ 96%
    sass:dist       14ms  ââ 2%
    Total 756ms

    npm ERR! weird error 6
    npm ERR! not ok code 0

So the `assets/components` directory was missing, weird. I know that Bower is installed, so what gives?

    $ bower install
    bower ESUDO         Cannot be run with sudo

    Additional error details:
    Since bower is a user command, there is no need to execute it with superuser permissions.
    If you're having permission errors when using bower without sudo, please spend a few minutes learning more about how your system should work and make any necessary repairs.

    http://www.joyent.com/blog/installing-node-and-npm
    https://gist.github.com/isaacs/579814

    You can however run a command with sudo using --allow-root option

Damn - that was a silent failure, or I didn't catch it during `npm install`. 

    $ bower install --allow-root
    ? May bower anonymously report usage statistics to improve the tool over time? No
    bower not-cached    git://github.com/zurb/bower-foundation.git#*
    bower resolve       git://github.com/zurb/bower-foundation.git#*
    bower download      https://github.com/zurb/bower-foundation/archive/5.5.2.tar.gz
    bower not-cached    git://github.com/FortAwesome/Font-Awesome.git#~4.4.0
    bower resolve       git://github.com/FortAwesome/Font-Awesome.git#~4.4.0
    bower download      https://github.com/FortAwesome/Font-Awesome/archive/v4.4.0.tar.gz
    bower extract       foundation#* archive.tar.gz
    bower resolved      git://github.com/zurb/bower-foundation.git#5.5.2
    bower not-cached    git://github.com/jquery/jquery.git#>= 2.1.0
    bower resolve       git://github.com/jquery/jquery.git#>= 2.1.0
    bower not-cached    git://github.com/mathiasbynens/jquery-placeholder.git#~2.0.7
    bower resolve       git://github.com/mathiasbynens/jquery-placeholder.git#~2.0.7
    bower not-cached    git://github.com/Modernizr/Modernizr.git#>= 2.7.2
    bower resolve       git://github.com/Modernizr/Modernizr.git#>= 2.7.2
    bower not-cached    git://github.com/carhartl/jquery-cookie.git#~1.4.0
    bower resolve       git://github.com/carhartl/jquery-cookie.git#~1.4.0
    bower not-cached    git://github.com/ftlabs/fastclick.git#>=0.6.11
    bower resolve       git://github.com/ftlabs/fastclick.git#>=0.6.11
    bower download      https://github.com/mathiasbynens/jquery-placeholder/archive/v2.0.9.tar.gz
    bower download      https://github.com/carhartl/jquery-cookie/archive/v1.4.1.tar.gz
    bower download      https://github.com/ftlabs/fastclick/archive/v1.0.6.tar.gz
    bower extract       fontawesome#~4.4.0 archive.tar.gz
    bower download      https://github.com/Modernizr/Modernizr/archive/v3.0.0.tar.gz
    bower download      https://github.com/jquery/jquery/archive/2.1.4.tar.gz
    bower extract       jquery.cookie#~1.4.0 archive.tar.gz
    bower extract       jquery-placeholder#~2.0.7 archive.tar.gz
    bower extract       fastclick#>=0.6.11 archive.tar.gz
    bower extract       modernizr#>= 2.7.2 archive.tar.gz
    bower extract       jquery#>= 2.1.0 archive.tar.gz
    bower mismatch      Version declared in the json (2.0.8) is different than the resolved one (2.0.9)
    bower resolved      git://github.com/mathiasbynens/jquery-placeholder.git#2.0.9
    bower not-cached    git://github.com/jquery/jquery.git#>=1.6
    bower resolve       git://github.com/jquery/jquery.git#>=1.6
    bower download      https://github.com/jquery/jquery/archive/2.1.4.tar.gz
    bower resolved      git://github.com/carhartl/jquery-cookie.git#1.4.1
    bower not-cached    git://github.com/jquery/jquery.git#>=1.2
    bower resolve       git://github.com/jquery/jquery.git#>=1.2
    bower download      https://github.com/jquery/jquery/archive/2.1.4.tar.gz
    bower resolved      git://github.com/ftlabs/fastclick.git#1.0.6
    bower resolved      git://github.com/jquery/jquery.git#2.1.4
    bower extract       jquery#>=1.2 archive.tar.gz
    bower extract       jquery#>=1.6 archive.tar.gz
    bower invalid-meta  modernizr is missing "main" entry in bower.json
    bower invalid-meta  modernizr is missing "ignore" entry in bower.json
    bower resolved      git://github.com/Modernizr/Modernizr.git#3.0.0
    bower resolved      git://github.com/jquery/jquery.git#2.1.4
    bower resolved      git://github.com/jquery/jquery.git#2.1.4
    bower resolved      git://github.com/FortAwesome/Font-Awesome.git#4.4.0
    bower install       foundation#5.5.2
    bower install       jquery-placeholder#2.0.9
    bower install       jquery.cookie#1.4.1
    bower install       fastclick#1.0.6
    bower install       jquery#2.1.4
    bower install       modernizr#3.0.0
    bower install       fontawesome#4.4.0

    foundation#5.5.2 assets/components/foundation
    âââ fastclick#1.0.6
    âââ jquery#2.1.4
    âââ jquery-placeholder#2.0.9
    âââ jquery.cookie#1.4.1
    âââ modernizr#3.0.0

    jquery-placeholder#2.0.9 assets/components/jquery-placeholder
    âââ jquery#2.1.4

    jquery.cookie#1.4.1 assets/components/jquery.cookie
    âââ jquery#2.1.4

    fastclick#1.0.6 assets/components/fastclick

    jquery#2.1.4 assets/components/jquery

    modernizr#3.0.0 assets/components/modernizr

    fontawesome#4.4.0 assets/components/fontawesome
    [root@develapache FoundationPress]# npm run build

    > foundationpress@1.4.0 build /var/www/clients/client0/web42/web/wp-content/themes/FoundationPress
    > grunt build

    Running "copy:scripts" (copy) task
    Created 1 directories, copied 5 files

    Running "copy:iconfonts" (copy) task
    Created 1 directories, copied 6 files

    Running "string-replace:fontawesome" (string-replace) task

    Running "sass:dist" (sass) task

    Running "concat:dist" (concat) task
    File "assets/javascript/foundation.js" created.

    Running "uglify:dist" (uglify) task
    File "assets/javascript/foundation.js" created.

    Done, without errors.


    Execution Time (2015-09-23 19:58:13 UTC)
    loading tasks  647ms  ââââââââââââââââââ 23%
    sass:dist      763ms  âââââââââââââââââââââ 27%
    concat:dist     33ms  â 1%
    uglify:dist     1.3s  âââââââââââââââââââââââââââââââââââ 47%
    Total 2.8s

Tl;dr: Bower as root is a no-no!

**Comment by olefredrik** [comment]
:+1:

**Comment by vercic** [comment]
I'm under Windows. I wan to copy all folder/files into my local manually and  lunch the theme. Can you put here all folders and files please in one zip file..?? Thank you.
